eZ/Publish/Core/Repository/SiteAccessAware/ContentService.php

```diff
@@ -129,9 +129,13 @@ public function deleteContent(ContentInfo $contentInfo)
         return $this->service->deleteContent($contentInfo);
     }
 
-    public function createContentDraft(ContentInfo $contentInfo, VersionInfo $versionInfo = null, User $user = null)
-    {
-        return $this->service->createContentDraft($contentInfo, $versionInfo, $user);
+    public function createContentDraft(
+        ContentInfo $contentInfo,
+        VersionInfo $versionInfo = null,
+        User $creator = null,
+        ?Language $language = null
```

Comment:
Please check if this change requires SA-aware unit tests coverage (AFAIR sth needs to be changed there as well)